styles.css

```diff
@@ -0,0 +1,121 @@
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
- [ ] For this project you are supposed to name the styles file called `index.css`. Please change your styles file name to `index.css` from `style.css`. 

### See the image for reference

![image](https://user-images.githubusercontent.com/83803414/160465159-f1e435da-8740-4149-86e8-bf10b804fdfc.png)

Reply:
Done